WebApplication/Backend/Repositorys/BuildingRepository.cs

```diff
@@ -3,11 +3,11 @@
 using System.Collections.Generic;
 using HealthClinicBackend.Backend.Model.Hospital;
 using HealthClinicBackend.Backend.Repository.DatabaseSql;
-using WebApplication.Backend.Repositorys.Interfaces;
+using HealthClinicBackend.Backend.Repository.Generic;
 
 namespace WebApplication.Backend.Repositorys
 {
-    public class BuildingRepository : IBuildingRepository
+    public class BuildingRepository : IGenericRepository<Building>
     {
         private readonly BuildingDatabaseSql _buildingRepository;
```

Comment:
This class already implements all those methods, I've added it here so I wouldn't have to refactor the whole web and hospital editor project